### PR TITLE
Add DRM support

### PR DESCRIPTION
**Pull request by kononenkoAnton**
Add support of DRM.
Parser from Zapp Entry to Theoplayer DRM API.
Data taken from Zapp Entry from path `exstensions/drm`

### CODE REVIEW

**Review by ranmeirman — CHANGES_REQUESTED** [review]
The pr has no description, please make sure all prs include description

File: plugins/zapp-react-native-theo-player/src/Services/DRM/index.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  if (fairplay) {
+    retVal["fairplay"] = {
+      licenseAcquisitionURL: fairplay?.license_url,
+      certificate_url: fairplay?.certificate_url,
+    };
+  }
+
+  const playreadyCerteficateURL = drmData?.playready?.license_url;
+  if (playreadyCerteficateURL) {
+    retVal["playready"] = { licenseAcquisitionURL: playreadyCerteficateURL };
```
Comment (ranmeirman) [inline plugins/zapp-react-native-theo-player/src/Services/DRM/index.ts:33]: I think we will use `license_url`, @inckie can tell accurately

File: plugins/zapp-react-native-theo-player/src/Services/DRM/index.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  }
+
+  const widevineCerteficateURL = drmData?.widevine?.license_url;
+  if (widevineCerteficateURL) {
+    retVal["widevine"] = { licenseAcquisitionURL: playreadyCerteficateURL };
```
Comment (ranmeirman) [inline plugins/zapp-react-native-theo-player/src/Services/DRM/index.ts:38]: Same as fairplay

**Review by ranmeirman — CHANGES_REQUESTED** [review]

File: plugins/zapp-react-native-theo-player/src/Services/DRM/index.ts (hunk 1 above)
Comment (kononenkoAnton) [reply plugins/zapp-react-native-theo-player/src/Services/DRM/index.ts:33]: @ranmeirman retVal this mapping to TheoPlayer Expectation. 
Line 31 get from our entry

File: plugins/zapp-react-native-theo-player/src/Services/DRM/index.ts (hunk 2 above)
Comment (kononenkoAnton) [reply plugins/zapp-react-native-theo-player/src/Services/DRM/index.ts:38]: @ranmeirman retVal this mapping to TheoPlayer Expectation. 
Line `36` get from our entry

File: plugins/zapp-react-native-theo-player/src/Services/DRM/index.ts (hunk 1 above)
Comment (inckie) [reply plugins/zapp-react-native-theo-player/src/Services/DRM/index.ts:33]: @Ran we did not change naming from official theo documentation, instead we convert structure in JS when pass from our entry to theirs

**Review by ranmeirman — APPROVED** [review]

**Pull request merged**